templates/password-enhancements-logged-out-message.html.twig

```diff
@@ -0,0 +1,12 @@
+{#
+/**
+ * @file
+ * Layout for showing a message for the logged out users.
+ *
+ * Available variable:
+ *   - message (string): The logout reason.
+ */
+#}
+<div id="logged-out-message" class="alert alert-danger">
```

Comment:
Same here, it's better to create an Attributes object that can be easily altered.